src/test/java/org/wso2/carbon/connector/integration/test/Epic/EpicConnectorIntegrationTest.java

```diff
@@ -0,0 +1,283 @@
+import org.wso2.connector.integration.test.base.RestResponse;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.io.IOException;
+/**
+ * Integration test for Epic Connector
+ */
+public class EpicConnectorIntegrationTest extends ConnectorIntegrationTestBase {
+
+    private Map<String, String> eiRequestHeadersMap = new HashMap<String, String>();
+    private Map<String, String> apiRequestHeadersMap = new HashMap<String, String>();
+
+    @BeforeClass(alwaysRun = true)
+    public void setEnvironment() throws Exception {
+        String connectorName = System.getProperty("connector_name") + "-connector-" +
+                System.getProperty("connector_version") + ".zip";
+        init(connectorName);
+        eiRequestHeadersMap.put("Accept-Charset", "UTF-8");
+        eiRequestHeadersMap.put("Accept", "application/json+fhir");
+        eiRequestHeadersMap.put("Content-Type", "application/json+fhir");
+        apiRequestHeadersMap.putAll(eiRequestHeadersMap);
+    }
+
+    /**
+     * Positive test case for getConformance method with mandatory parameters.
+     */
+    @Test(enabled = true, description = "epic {getConformance} integration test with mandatory parameters.")
+    public void getConformanceWithMandatoryParameters() throws IOException, JSONException {
+        eiRequestHeadersMap.put("Action", "urn:getConformance");
+        RestResponse<JSONObject> eiRestResponse = sendJsonRestRequest(proxyUrl, "POST", eiRequestHeadersMap, "getConformanceMandatory.json");
+        final String apiEndPoint = connectorProperties.getProperty("base") + "/metadata";
+        RestResponse<JSONObject> apiRestResponse = sendJsonRestRequest(apiEndPoint, "GET", apiRequestHeadersMap);
+        Assert.assertEquals(eiRestResponse.getHttpStatusCode(), 200,"Invalid parameter value");
+        Assert.assertEquals(apiRestResponse.getHttpStatusCode(), 200, "Invalid parameter value");
+    }
+
+    /**
+     * Positive test case for readSpecificResourceById method with Mandatory parameters.
+     */
+    @Test(enabled = true, description = "epic {readSpecificResourceById} integration test with Mandatory parameters.")
+    public void readSpecificResourceByIdWithMandatoryParameters() throws IOException, JSONException {
+        eiRequestHeadersMap.put("Action", "urn:readById");
+        RestResponse<JSONObject> eiRestResponse = sendJsonRestRequest(proxyUrl, "POST", eiRequestHeadersMap, "readSpecificResourceById.json");
+        final String apiEndPoint = connectorProperties.getProperty("base") + "/"
+                + connectorProperties.getProperty("type") + "/" + connectorProperties.getProperty("id");
+        RestResponse<JSONObject> apiRestResponse = sendJsonRestRequest(apiEndPoint, "GET", apiRequestHeadersMap);
+        Assert.assertEquals(eiRestResponse.getHttpStatusCode(), 200,"Invalid parameter value");
+        Assert.assertEquals(apiRestResponse.getHttpStatusCode(), 200, "Invalid parameter value");
```

Comment:
Can we assert more properties from response body ? fix other test cases as well.

refer [1].
[1] https://github.com/wso2-extensions/esb-connector-salesforcerest/blob/master/src/test/java/org/wso2/carbon/connector/integration/test/salesforcerest/SalesforceRestConnectorIntegrationTest.java#L118